### DISCUSSION TITLE
FILDevSummit Singapore&Iceland'23: Down to the Core: Filecoin Protocol Development & Evolution

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#662] [General]
> What's FILDev Summit ➡️ [here](https://github.com/filecoin-project/community/discussions/661).

**Track Leads:** @[jennijuju](https://twitter.com/_jennijuju) & @nicola 
Time to discuss deeper how the Filecoin protocol can further evolve, especially with the introduction of FVM! Things to discuss: Filecoin layer 0 vs 1 vs 2 protocol, moving functionality into FVM, IPC regions for Filecoin, upgrade path, Better PoReps (proof of space & IPC), new consensus, upgrading the filecoin economy & etcc.. Expecting reps from Filecoin core implementers, CNL, CEL, CL, and active FIP authors to participate and share their 🧠 in this track!

Full agenda ⏭️ [fildev.io](https://fildev.io/)
Subscribe this discussion to get updates from this track!

**Comment by momack2** [comment]
<img width="583" alt="CleanShot 2023-09-14 at 16 45 41@2x" src="https://github.com/filecoin-project/community/assets/618519/c8a1250e-42f9-4584-a919-0908070c01fa">
